docs/days/day-12.md

```diff
@@ -1,5 +1,70 @@
 # Jobeet Day 12: The API
 
+When you post a job, you will want to have the greatest exposure possible.
+If your job is syndicated on a lot of small websites, you will have a better chance to find the right person. That's the power of the [long tail][1].
```

Comment:
Could it be `If your job advertisement is spread across multiple websites, you have a better chance ...`   :blush: